src/app/collection/pages/collection-502/collection-502.component.ts

```diff
@@ -7,9 +12,45 @@ import { Component, OnInit } from '@angular/core';
 })
 export class Collection502Component implements OnInit {
 
-  constructor() { }
+  learningObjects: LearningObject[];
+  guidelineNames: [];
+  loading = true;
+  query = {
+    limit: 5,
+    collection: '502_project'
+  };
+
+  constructor(
+    private navbarService: NavbarService,
+    private learningObjectService: LearningObjectService
+  ) { }
+
+  async ngOnInit() {
+
+    this.navbarService.show();
+    this.fetchLearningObjects(this.query);
```

Comment:
```suggestion
    await this.fetchLearningObjects(this.query);
```
Since it is an async function

Reply:
I totally missed that one. Thank you!